Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -291,10 +291,15 @@ void Update()
                     if (!buildingUnlocked && buildingType < DFLocation.BuildingTypes.Temple
                         && buildingType != DFLocation.BuildingTypes.HouseForSale)
                     {
-                        string storeClosedMessage = HardStrings.storeClosed;
-                        storeClosedMessage = storeClosedMessage.Replace("%d1", openHours[(int)buildingType].ToString());
-                        storeClosedMessage = storeClosedMessage.Replace("%d2", closeHours[(int)buildingType].ToString());
-                        DaggerfallUI.Instance.PopupMessage(storeClosedMessage);
+                        string buildingClosedMessage;
```

Comment:
For brevity you could write this like:

`
string buildingClosedMessage = (buildingType == DFLocation.BuildingTypes.GuildHall) ? HardStrings.guildClosed : HardStrings.storeClosed;
`

Reply:
Of course I'll change it. I was never fond of the ternary operator syntax, not very readable in my opinion, that's why I tend to never use it :)

Reply:
I find it a lot more readable when instantiating a var to one of two different values, but it's my personal preference so that's why it was a suggestion only. :)